dev/com.ibm.ws.grpc_fat/bnd.bnd

```diff
@@ -61,7 +61,8 @@ tested.features:\
     pages-4.0,\
     mprestclient-4.0,\
     mpopenapi-4.0,\
-    restfulws-4.0
+    restfulws-4.0,\
+    noship-1.0
```

Comment:
Remove this here: https://github.com/OpenLiberty/open-liberty/issues/29565. The noship-1.0 feature was added because `appsecurity-6.0 and jwtsso-1.0` end up pulling in `io.openliberty.jwtSso1.0.internal.ee-11.0` which is still noship